package.json

```diff
@@ -3,20 +3,6 @@
   "version": "1.0.0",
   "description": "shareable eslint config for merit app",
   "license": "MIT",
-  "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.28.1",
-    "eslint": "^7.30.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-functional": "^3.2.1",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.24.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-sort-destructure-keys": "^1.3.5",
-    "eslint-plugin-sort-imports-es6-autofix": "^0.6.0",
-    "eslint-plugin-sort-keys-fix": "^1.1.1",
-    "prettier": "^2.3.2",
-    "typescript": "^4.3.5"
```

Comment:
@Stephen2  removing `peerDependencies` will not have any functional effects but we need `peerDependencies` as it suggests which dependencies we need to install in our project